test/foundry/util/TestBase.sol

```diff
@@ -42,6 +42,16 @@ contract TestBase is Test {
         return array;
     }
 
+    function _generateUint256Array(
```

Comment:
We don't need to add this function if we generate all three arrays in the same for-loop.